specs/CBOR_SETS.md

```diff
@@ -0,0 +1,29 @@
+# Sets for CBOR
+
+This document specifies a tag for sets in Concise Binary Object Representation (CBOR) [1].
+
+    Tag: 258
+    Data item: array
+    Semantics: Set
+    Point of contact: Input Output HK <info@iohk.io>
+    Description of semantics: https://github.com/input-output-hk/cardano-sl/tree/master/specs/CBOR_SETS.md
+
+## Semantics
+
+Tag 258 can be applied to an array to indicate that the tagged object is a set. Sets should be handled
+similarly to maps: a set that has duplicate elements may be well-formed, but it is not valid.
+Other constraints on map keys [2] apply to set elements as well.
```

Comment:
Is it worth being explicit here rather than only referring?

Using the same wording as the RFC for maps, as you've done, is sensible.